openqa_review/openqa_review.py

```
@@ -619,8 +621,7 @@ def __init__(self, bugref, bugref_href, query_issue_status=False, progress_brows
                     self.subject = self.json['subject']
                     self.priority = self.json['priority']['name']
                     self.last_comment_date = datetime.datetime.strptime(self.json['updated_on'], "%Y-%m-%dT%H:%M:%SZ")
-                # bugref.startswith('bsc#') or bugref.startswith('boo#')
-                else:
```

Please don't delete else-branch. It's good programming practice to always have an else-branch. Could be that you want to throw an error in case of else

---

Added 'else' branch. And also changed if-else to ternary form.